src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobBuilder.cs

```diff
@@ -788,7 +788,7 @@ public void WriteBytes(byte[] buffer, int start, int byteCount)
             WriteBytes(buffer.AsSpan(start, byteCount));
         }
 
-        private void WriteBytes(ReadOnlySpan<byte> buffer)
+        internal void WriteBytes(ReadOnlySpan<byte> buffer)
```

Comment:
move below public methods?

Reply:
I prefer to not move it to minimize changes. It will become public with #85280 either way.